src/components/List/ListItem.tsx

```diff
@@ -35,6 +46,7 @@ class ListItem extends UIComponent<Extendable<IListItemProps>, any> {
 
   static propTypes = {
     as: customPropTypes.as,
+    focusableItemProps: PropTypes.object,
```

Comment:
nit: maybe we should place it closer to `accessibility` prop

Reply:
done